source/code/go/src/plugins/oms.go

```diff
@@ -958,6 +978,13 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 		Log("ResourceID=%s", ResourceID)
 		Log("ResourceName=%s", ResourceName)
 	}
+	
+	containerRuntime = os.Getenv(ContainerRuntimeEnv)
```

Comment:
init containerRuntime to docker (default). Then get from os.Getenv, and assign it to containerRuntime only if its not empty.

Reply:
done.